src/hal/lv_hal_tick.c

```diff
@@ -29,8 +30,13 @@
  *  STATIC VARIABLES
  **********************/
 #if !LV_TICK_CUSTOM
-    static uint32_t sys_time = 0;
-    static volatile uint8_t tick_irq_flag;
+    #if LV_USE_ATOMICS == 1
+        static _Atomic(uint32_t) sys_time = 0;
+        static _Atomic(uint8_t) tick_irq_flag;
```

Comment:
I'd use `atomic_int` and `volatile int` just to use the same pattern as with `flushing` and `flushing_last`.

Reply:
Changed that :)